# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoader.cpp
# Commit: d77d76d37d4d
# Full Hash: d77d76d37d4dccf39c12fa81c970088b5388d0f1
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-05-13 08:22:56
# Regressor Bug: 1518919
# File Overlap Count: 1
# Description:
#   Bug 1518919 - Propagate CSS visibility information to descendant documents via IPC call for fission. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30460
# ==============================================================================

diff -r 02b3eb8a1a13 -r d77d76d37d4d dom/base/nsFrameLoader.cpp
--- a/dom/base/nsFrameLoader.cpp	Thu May 09 07:37:03 2019 +0000
+++ b/dom/base/nsFrameLoader.cpp	Mon May 13 01:49:07 2019 +0000
@@ -2518,6 +2518,16 @@
   return NS_OK;
 }
 
+void nsFrameLoader::SendIsUnderHiddenEmbedderElement(
+    bool aIsUnderHiddenEmbedderElement) {
+  MOZ_ASSERT(IsRemoteFrame());
+
+  if (mBrowserBridgeChild) {
+    mBrowserBridgeChild->SetIsUnderHiddenEmbedderElement(
+        aIsUnderHiddenEmbedderElement);
+  }
+}
+
 void nsFrameLoader::UpdateBaseWindowPositionAndSize(
     nsSubDocumentFrame* aIFrame) {
   nsCOMPtr<nsIBaseWindow> baseWindow = GetDocShell(IgnoreErrors());